core/src/main/java/org/apache/spark/memory/MemoryConsumer.java

```diff
@@ -60,8 +60,6 @@ protected long getUsed() {
 
   /**
    * Force spill during building.
-   *
```

Comment:
It's not "for testing only". I'd even say that it's more often used in a non-test code than test code. That made the comment no longer correct. See `ShuffleExternalSorter` and `UnsafeShuffleWriter` which both are `ShuffleWriter` instances.